bitnami/kafka/templates/statefulset.yaml

```diff
@@ -155,6 +155,8 @@ spec:
           value: {{ .Values.logRetentionCheckIntervalMs | quote }}
         - name: KAFKA_LOG_RETENTION_HOURS
           value: {{ .Values.logRetentionHours | quote }}
+        - name: KAFKA_LOG_MESSAGE_FORMAT_VERSION
+          value: {{ .Values.logMessageFormatVersion | replace "_" "" | quote }}
```

Comment:
You don't need the replace thing here. That was just a workaround to avoid helm parsing big numbers in values.yaml as scientific notation.